### PR TITLE
Validate location isn't blank

### PR DESCRIPTION
**Pull request by mveytsman**
No more trips to [Null Island](https://en.wikipedia.org/wiki/Null_Island)

### CODE REVIEW

File: config/runtime.exs [hunk 1]
```diff
@@ -3,11 +3,80 @@ import Config
...
+    end
+
+    case System.get_env("MAILCHIMP_API_KEY") do
+      nil ->
+        config :bike_brigade, :mailchimp, adapter: BikeBrigade.MailchimpApi.FakeMailchimp
+
+      key ->
+        config :bike_brigade, :mailchimp, adapter: BikeBrigade.MailchimpApi.Http
+        config :mailchimp, api_key: key
+    end
```
Comment (teesloane) [inline config/runtime.exs:79]: will a rebase on main remove these from the diff?

Comment (mveytsman) [reply config/runtime.exs]: Good catch thanks!

**Review by teesloane — APPROVED** [review]
it looks .... good to me?

**Comment by mveytsman** [comment]
Closes #227

**Pull request merged**